project/WithDebugCommand.scala

```diff
@@ -85,7 +85,8 @@ object WithDebugCommand {
           Seq(DEBUG_OPTION)
         else Seq()
       val javaOpts: Seq[String] = Seq(
-        truffleNoBackgroundCompilationOptions,
+        if (debugFlags.contains(debuggerOption)) Seq()
```

Comment:
Allowing Truffle background compilation when a debugger is attached? Why? Doesn't this make debugging more difficult and less deterministic? Was the debugging too slow?

Reply:
- yes, debugging is then too slow
- some projects `runtime-language-epb/test` doesn't have a support for _disabling background compilation_
- disabling background compilation is _unnatural_
- regular truffle execution runs with _background compilation_